examples/custom_file_handler/custom_file_handler.ts

```diff
@@ -5,7 +5,7 @@
 // import { WebUI } from "../../mod.ts";
 
 // To import from online package registry (Production)
-import { WebUI } from "jsr:@webui/deno-webui@2.5.9"; // import {WebUI} from "https://deno.land/x/webui@2.5.9/mod.ts";
+import { WebUI } from "@webui/deno-webui"; // import {WebUI} from "https://deno.land/x/webui@2.5.9/mod.ts";
```

Comment:
I think this is better, so we don't have to update each time there is a new version